island-is/island.is
# feat(samradsgatt): Kam 806 - add props to case details page and refactor

# Add props to case details page and refactor

https://veflausnir.atlassian.net/browse/KAM-806
https://veflausnir.atlassian.net/browse/KAM-844

## What

I added props to necessary components on the case details page.
I made components standalone and make the code more readable  by removing the left, middle and right side components.

## Why

Props missing and too complicated code.

## Screenshots / Gifs

<img width="1415" alt="image" src="https://user-images.githubusercontent.com/106164521/216026219-b9834787-86e5-4bff-b0d6-cea813e89a84.png">

https://www.figma.com/file/gFSUsFCIMYWRGa94XsmhpT/DEV-Samr%C3%A1%C3%B0sg%C3%A1tt?node-id=246%3A8033&t=HPNmkoygoUxHVOlR-0

## How to test
Checkout branch feature/samradsgatt/KAM-806-props-add-props-to-case-details-page
Run the code locally
Navigate to http://localhost:4200/samradsgatt/mal/id
Compare desing and the page.
Look at the code and make sure it makes sense :)

## Checklist:
